src/main/java/competition/subsystems/lights/LightsCommunicationSubsystem.java

```diff
@@ -46,7 +47,9 @@ public enum LightsStateMessage {
         RobotNotBooted(31),
         RobotDisabled(1),
         Enabled(2),
-        GamePieceCollected(3);
+        GamePieceCollected(3),
+        ArmAtTargetPosition(4);
```

Comment:
You'll have to merge from `main`, I just slipped a new state in here already so you'll have to make yours `5`